packages/world/ts/register/typeExtensions.ts

```diff
@@ -1,4 +1,8 @@
-import { WorldConfig, WorldUserConfig } from "../library";
+import { OrDefaults } from "@latticexyz/common/type-utils";
+import { MUDCoreUserConfig } from "@latticexyz/config";
+import { ExpandSystemsConfig, WorldConfig, WorldConfigDefaults, WorldUserConfig } from "../library";
```

Comment:
Here we import from `library`, and `library/config/index.ts` imports from here. Does this circular import cause issues? Does it matter in which order the type extensions of store and world are imported? (I can't tell for sure which one would be imported first now, but I guess still `store` because it is listed before the type definition below?)

Reply:
unlike `store`, this wasn't really a problem in `world`. https://github.com/latticexyz/mud/pull/752#discussion_r1188523956 removes any circular imports there